torch_audiomentations/utils/file.py

```diff
@@ -0,0 +1,21 @@
+import os
+import glob
+import librosa
+
+
+SUPPORTED_EXTENSIONS = ['.wav']
+
+
+def find_files(path):
```

Comment:
The name of the function doesn't seem specific enough. `find_audio_files` maybe?